astropy/constants/tests/test_prior_version.py

```diff
@@ -59,7 +60,10 @@ def test_e():
     assert isinstance(e.gauss, Q)
     assert isinstance(e.esu, Q)
 
-    assert e.si * E == Q(100, 'eV/m')
+    # Cannot use == comparison here because conversion to eV
+    # uses constants.si.e value that can vary over time.
```

Comment:
I was confused by the comment: maybe "that is taken from the current set of constants, which is not the same as that of astropy 1.3"

Reply:
The real issue is that the definition of E above was "cooked up" based on the ratio of `e` from `astropyconst13` and `astropyconst20`.

```
import astropy.units as u
from astropy.constants.astropyconst13 import e as e_13
from astropy.constants.astropyconst20 import e as e_20
from astropy.constants.astropyconst40 import e as e_40

(e_20.si / e_13.si).value
# 1.0000000348276221
(e_40.si / e_13.si).value
# 1.0000000430664144
```
If the `e_40 / e_13` ratio were used above in the definition of `E`, then this test could be retained. I am not sure if it is a meaningful test though.

Reply:
@stargaser , even if I use a ratio as you suggested, the test would still have to be updated every time the default changes. Is that desirable?

Reply:
My feeling is that we should drop this part of this particular test (everything that involves `E`, including its definition).

Reply:
Agreed, it is no longer clear what we're testing if we need to add work-arounds.

Reply:
I am not sure about dropping `E` altogether, as some part of the test that uses it still works. But I dropped the controversial `assert e.si * E == Q(100, 'eV/m')` that was failing.